<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SystemMainWin</class>
 <widget class="QWidget" name="SystemMainWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1051</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="leftWidget" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>100</width>
     <height>450</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="headLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/images/head.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="logoutBtn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>400</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>200</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(239, 41, 41);</string>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="videoRecordtoolButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>100</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>视频记录</string>
    </property>
   </widget>
   <widget class="QLabel" name="nameLabel">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>100</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 13pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string>陈某人</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QToolButton" name="drivingModetoolButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>100</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>行车模式</string>
    </property>
   </widget>
   <widget class="QToolButton" name="reverseModetoolButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>220</y>
      <width>100</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>倒车模式</string>
    </property>
   </widget>
   <widget class="QToolButton" name="birdViewtoolButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>260</y>
      <width>100</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>鸟瞰模式</string>
    </property>
   </widget>
   <widget class="QToolButton" name="featureRecordtoolButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>300</y>
      <width>100</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>特征记录</string>
    </property>
   </widget>
   <widget class="QToolButton" name="systemSettingtoolButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>340</y>
      <width>100</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>系统设置</string>
    </property>
   </widget>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>40</y>
     <width>850</width>
     <height>450</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>5</number>
   </property>
   <widget class="QWidget" name="videoRecordPage">
    <property name="styleSheet">
     <string notr="true">background-color: rgb(238, 238, 236);</string>
    </property>
    <widget class="QListWidget" name="video_listWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-10</y>
       <width>850</width>
       <height>450</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="iconSize">
      <size>
       <width>412</width>
       <height>235</height>
      </size>
     </property>
     <property name="movement">
      <enum>QListView::Static</enum>
     </property>
     <property name="resizeMode">
      <enum>QListView::Adjust</enum>
     </property>
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="gridSize">
      <size>
       <width>416</width>
       <height>235</height>
      </size>
     </property>
     <property name="viewMode">
      <enum>QListView::IconMode</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="drivingModePage">
    <widget class="QLabel" name="drivingVideo">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>850</width>
       <height>450</height>
      </rect>
     </property>
     <property name="text">
      <string>行车模式</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="reverseModePage">
    <widget class="QLabel" name="reverseVideo">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>850</width>
       <height>450</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#reverseVideo{
	background-color: rgb(173, 127, 168);
}</string>
     </property>
     <property name="text">
      <string>倒车模式</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="birdEyeMode">
    <widget class="QLabel" name="birdEyeLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>300</width>
       <height>475</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#birdEyeLabel{
	background-color: rgb(138, 226, 52);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resources.qrc">:/images/bg.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="realTimeLabel">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>0</y>
       <width>550</width>
       <height>475</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(233, 185, 110);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="featureRecordPage">
    <widget class="QListWidget" name="pic_listWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>850</width>
       <height>450</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="midLineWidth">
      <number>3</number>
     </property>
     <property name="iconSize">
      <size>
       <width>415</width>
       <height>235</height>
      </size>
     </property>
     <property name="movement">
      <enum>QListView::Static</enum>
     </property>
     <property name="resizeMode">
      <enum>QListView::Adjust</enum>
     </property>
     <property name="gridSize">
      <size>
       <width>416</width>
       <height>235</height>
      </size>
     </property>
     <property name="viewMode">
      <enum>QListView::IconMode</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="settingPage">
    <property name="styleSheet">
     <string notr="true">#settingPage
{background-color: rgb(18, 25, 115);
}</string>
    </property>
    <widget class="QLabel" name="title_label">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>10</y>
       <width>491</width>
       <height>59</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(238, 238, 236);
font: 30pt &quot;Ubuntu&quot;;</string>
     </property>
     <property name="text">
      <string>360°智能行车辅助系统设置</string>
     </property>
    </widget>
    <widget class="QLabel" name="iplabel">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>100</y>
       <width>339</width>
       <height>59</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 20pt &quot;Ubuntu&quot;;
color: rgb(238, 238, 236);</string>
     </property>
     <property name="text">
      <string>服务器系统IP：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>240</y>
       <width>33</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(238, 238, 236);
font: 15pt &quot;Ubuntu&quot;;</string>
     </property>
     <property name="text">
      <string>GB</string>
     </property>
    </widget>
    <widget class="QLabel" name="portlabel_3">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>240</y>
       <width>211</width>
       <height>49</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 20pt &quot;Ubuntu&quot;;
color: rgb(238, 238, 236);</string>
     </property>
     <property name="text">
      <string>本地存储大小：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="memoryEdt">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>250</y>
       <width>250</width>
       <height>40</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#port_lineEdit{
	border-radius:3px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="portlabel">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>170</y>
       <width>243</width>
       <height>59</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 20pt &quot;Ubuntu&quot;;
color: rgb(238, 238, 236);</string>
     </property>
     <property name="text">
      <string>服务器端口号：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="port_lineEdit">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>180</y>
       <width>250</width>
       <height>40</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#port_lineEdit{
	border-radius:3px;
}</string>
     </property>
    </widget>
    <widget class="QLabel" name="portlabel_5">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>310</y>
       <width>211</width>
       <height>39</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 20pt &quot;Ubuntu&quot;;
color: rgb(238, 238, 236);</string>
     </property>
     <property name="text">
      <string>存满自动删除</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>313</y>
       <width>25</width>
       <height>37</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>380</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 24pt &quot;Ubuntu&quot;;</string>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QFrame" name="bottomLayerFrame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1051</width>
     <height>521</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/background.png);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <zorder>bottomLayerFrame</zorder>
  <zorder>leftWidget</zorder>
  <zorder>stackedWidget</zorder>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
